app/src/main/java/com/example/gdsc_assign/HomeFragment.kt

```diff
@@ -36,12 +40,23 @@ class HomeFragment : Fragment() {
     }
     fun initialize(){
         with(todolist){
-            add(ToDo("11월27일", "안드로이드 스터디 과제하기", false))
-            add(ToDo("11월27일", "안드로이드 자기소개 구경하기", true))
-            add(ToDo("11월27일", "안드로이드 2주차 공부하기", true))
+            add(ToDo("11월27일", "안드로이드 스터디 과제하기", true))
+            add(ToDo("11월27일", "안드로이드 자기소개 구경하기", false))
+            add(ToDo("11월27일", "안드로이드 2주차 공부하기", false))
            add(ToDo("11월27일", "안드로이드 1주차 복습하기", false))
-            add(ToDo("11월27일", "이슬톡톡 연마하기", true))
-            add(ToDo("11월27일", "살아남기..!", true))
+            add(ToDo("11월27일", "이슬톡톡 연마하기", false))
+            add(ToDo("11월27일", "살아남기..!", false))
+        }
+    }
+    fun  checkCompleteTodo() : Int{
+        var a = 0
+        for(i : Int in 0..todolist.size-1){
+            if(todolist.get(i).isDone == true){
+                a++
```

Comment:
```suggestion
       for(i : Int in 0..<todolist.size){
            if(todolist[i].isDone){
                a++
            }
        }
```
다음과같이 simplify할 수 있는 코드는 안드로이드 스튜디오 내부에서 초록색밑줄로 제안사항을 제공합니다 ~! 이를 한번씩 확인해보면 좋을 것 같아요